apps/sscs/sscs-evidence-share/demo.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
 spec:
   values:
     java:
-      image: hmctspublic.azurecr.io/sscs/evidence-share:pr-1246-c0c058d-20220321171145 #{"$imagepolicy": "flux-system:demo-sscs-evidence-share"}
+      image: hmctspublic.azurecr.io/sscs/evidence-share:pr-1261-c0c058d-20220321171145 #{"$imagepolicy": "flux-system:demo-sscs-evidence-share"}
```

Comment:
@MohammedHMCTS you don't need to change this line as the pr details are incorporated from demo-image-policy.yaml file